Quickref.md

```diff
@@ -0,0 +1,170 @@
+# Reflex Quick(ish) Reference
+
+## Typeclasses
+
+Many Reflex functions operate in monadic context `X a`, where the monadic type X supports various additional typeclasses such as MonadWidget, MonadHold, or MonadSample in addition to Monad itself.  For Reflex-Dom users, the type X will generally be `Widget Spider (Gui Spider (WithWebView SpiderHost) (HostFrame Spider))`, from the Reflex-Dom main entry point:
+
+```haskell
+mainWidget :: Widget Spider (Gui Spider (WithWebView SpiderHost) (HostFrame Spider)) () -> IO ()
```

Comment:
I think the point is that there's no need to refer to the type of mainWidget *at all* in the Reflex quickref.